README.md

```diff
@@ -14,18 +14,111 @@ All bugs, stories, and backlog for this project are managed through [Hyperledger
+ your application with libindy.       
+
+#### Release channels
+{release channels} pointed in previous section may be one of these values: 
+    
+* `master` - development builds for each push to master branch.
+* `rc` - release candidates.
+* `stable` - stable releases.
+
+Please refer to [release workflow](doc/release-workflow.md) for more details.
+
+### Install for MacOS        
+Now we haven't prebuild library in some shared place. You can build
+library yourself. Please refer to How-to-build section. 
+
+After build add to LD_LIBRARY_PATH and to DYLD_LIBRARY_PATH 
+environment variables path to builded library. It's necessary 
+for dynamic linkage your application with libindy. At first dynamic linker
+browse library in LD_LIBRARY_PATH, if library in your application doesn't include directory names.
+If library in your application include any directory name, then dynamic linker will search library
+in DYLD_LIBRARY_PATH(not LD_LIBRARY_PATH). So for reliability we recommend you set both this variables.
+            
+### Install for RHEL based distro (Amazon Linux 2017.03)           
+Now we haven't prebuild library in some shared place. You can build
+library yourself. Please refer to How-to-build section.
+
+After build add to LD_LIBRARY_PATH environment variable path to builded library. 
+It's necessary for dynamic linkage your application with libindy.
+
+## How to build
 
 * [Ubuntu based distro (Ubuntu 16.04)](doc/ubuntu-build.md)
 * [RHEL based distro (Amazon Linux 2017.03)](doc/rhel-build.md)
 * [Windows](doc/windows-build.md)
 * [MacOS](doc/mac-build.md)
 
+## How to start local nodes pool with docker
+
+Start local nodes pool on `127.0.0.1:9701-9708` with Docker:
+     
+     ```     
+     docker build -f ci/indy-pool.dockerfile -t indy_pool .
+     docker run -itd -p 9701-9708:9701-9708 indy_pool
+     ```     
+     
+ Dockerfile `ci/indy-pool.dockerfile` supports optional pool_ip param that allows 
+ changing ip of pool nodes in generated pool configuration. The following commands 
+ allow to start local nodes pool in custom docker network and access this pool 
+ by custom ip in docker network:
+     
+     ```
+     docker network create --subnet 10.0.0.0/8 indy_pool_network
+     docker build --build-arg pool_ip=10.0.0.2 -f ci/indy-pool.dockerfile -t indy_pool .
+     docker run -d --ip="10.0.0.2" --net=indy_pool_network indy_pool
+     ``` 
+ Note that for Windows and MacOS this approach have some issues. Docker for these OS run in
+ their virtual environment. First command creates network for container and host can't
+ get access to that network because container placed on virtual machine. You must appropriate set up 
+ networking on your virtual environment.
+
 ## Wrappers documentation
 * [.Net](wrappers/dotnet/README.md)
 * [Java](wrappers/java/README.md)
 * [Python](wrappers/python/README.md)
-* [iOS](wrappers/ios/ios-build.md)
+* [iOS](wrappers/ios/README.md)
```

Comment:
In fact we have 2 pod specs. 

- libindy and ObjectiveC wrapper. Let's discuss how to handle this.